index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title> my portfolio</title>
+</head>
+
+<body>
+    <main class="container">
+        <section>
+            <a class="logo">logos</a>
+            <nav class="humbuger"><img src="images/icones/Union.png" alt="humburger"> </nav>
+        </section>
+        <div class="contain">
+            <div class="container-title">
+                <h1 class="item1">Faustin Makutano</h1>
```

Comment:
- [ ] You should stick to the Figma design and use `font-size: 48px, color: #7D73FC;  font-weight: 700; `as in the figma design.
#### Design
![image](https://user-images.githubusercontent.com/44057863/170758901-b5c361cc-90d8-4328-adb1-d7f016361fd1.png)
#### Impl
![image](https://user-images.githubusercontent.com/44057863/170759069-bb1737d2-a670-4838-aa51-02f403a21d57.png)

- [ Optional ] You should also add some padding vertically.